Formula/mpd.rb

```diff
@@ -1,6 +1,7 @@
 class Mpd < Formula
   desc "Music Player Daemon"
   homepage "http://www.musicpd.org/"
+  revision 1
```

Comment:
Modified formulas are available as soon as they're pulled; the revision is to signal to `brew` that people who already have the formula installed should rebuild or reinstall it. Since this just adds a new build option, that shouldn't be necessary here, since rebuilding wouldn't change any existing installations.